### ISSUE TITLE
CRS year does not change- bug?

### ISSUE THREAD

**Issue by PetyaKangalova** [devinit/D-Portal#543]
@xriss  and @notshi , searching for a country 'Botswana' and then clicking on the "Where does the money come from?" brings me to this page:  http://d-portal.org/ctrack.html?country_code=BW#view=donors&year=2018

![image](https://user-images.githubusercontent.com/24607542/71722629-6ca87800-2e21-11ea-8ee1-319b49dda4bf.png)

Then moving the arrow to go back to previous years brings me to this page (http://d-portal.org/ctrack.html?country_code=BW#view=donors&year=2017).  The CRS value changes but the year still shows as 2017.

![image](https://user-images.githubusercontent.com/24607542/71722705-af6a5000-2e21-11ea-918d-405817f47c5e.png)

Basically, it looks to me that the CRS data changes when you go back and forward with the arrows but the year stays at 2017. Could you check whether this is a bug?

Thanks! @WendyMAThomas

**Comment by notshi** [comment]
The CRS column is a static column that shows the latest data / year we have for each publisher so it doesn't change when you click through the arrows (there's a footnote below that explains more).

It appears changed because of the order of the publishers - clicking through the arrows resets the table and sorts it by the middle IATI Spend column with the largest number at the top.

If you sort it by name, you'll see that the numbers don't change.

**Comment by PetyaKangalova** [comment]
@notshi , thank your response and confirming that:

**1) CRS column shows only 2017 data-** I somehow thought d-portal also shows 2016 and older CRS data. And yes, 2018 will soon be available so the column can be updated. We will let you know once the data is available.

**2) IATI spend table -** I did not pick up on this, but yes, your are right ;) The arrows indeed resets the table and sorts it by the middle IATI Spend column with the largest number at the top. The CRS data remains the same for each individual publishers.
